dbms/src/Common/Elf.h

```diff
@@ -6,9 +6,42 @@
 #include <optional>
 #include <functional>
 
+#ifdef __APPLE__
```

Comment:
But Apple uses Mach-O instead of ELF.
This file (along with: Dwarf, SymbolCache, PHDRCache) is totally useless on Mac.

Reply:
Elf.h is included from several files. It is easier to fix a header rather than all of them.

Reply:
Ok.